.stylelintrc.js

```diff
@@ -218,7 +218,7 @@ module.exports = {
     ],
 
     // Sass rules
-    'max-nesting-depth': 2,
+    'max-nesting-depth': 3,
```

Comment:
This might be an OK change but the [StyleGuide](https://docs.google.com/document/d/1U8DqwOyYlNweZSa5Kn_sJW9Kx079vMclYwVi8yyPYaY/) should be amended to include this.

Reply:
Good point – I've added a note to the style guide...

We need a nesting depth of 3 to make the most of SASS features e.g. looping over a map of class variants to create classes